src/include/dhcp-server/widgets.rb

```diff
@@ -8,6 +8,8 @@
 
 require "yast"
 require "ui/service_status"
+require "y2firewall/firewalld"
+require "y2firewall/helpers/interfaces"
```

Comment:
if you include the helpers the `firewalld` method will be already defined and also you could use the interface_zone(name)` method

https://github.com/yast/yast-yast2/blob/master/library/network/src/lib/y2firewall/helpers/interfaces.rb#L55